### ISSUE TITLE
Random animation of AnimList in warhead

### ISSUE THREAD

**Issue by NetsuNegi** [Phobos-developers/Phobos#197]
**Description**: Let AnimList animations display randomly, just like Custom ‘SplashList’ on Warheads.
**INI Code**: 
```ini
[SOMEWARHEAD]
AnimList.PickRandom=no ; play a random animation from the list? boolean, defaults to no

**Comment by secsome** [comment]
Just use 
``ini
[SOMEWARHEAD]
EMEffect=yes
```

**Comment by secsome** [comment]
Implemented in fba3c8b
